content/applications/finance/payment_acquirers/ogone.rst

```diff
@@ -5,84 +5,105 @@ Ogone
+:guilabel:`Checks for DirectLink and Batch (Automatic)`. You can leave the IP address field blank.
 
-SHA Key OUT
-~~~~~~~~~~~
+Your **SHA-IN** and **SHA-OUT** passphrases should be different, and between 16 and 32 characters
+long. Make sure to use the same **SHA-IN** and **SHA-OUT** passphrases throughout the entire Ogone
+configuration, as Odoo only allows a single **SHA-IN** and single **SHA-OUT** passphrase.
 
-In order to retrieve the SHA Key OUT, log into your ogone account, go to
-:menuselection:`Configuration --> Technical Information --> Transaction feedback --> All transaction
-submission modes`, and get or generate your **API Key** and **Client Key**. Be careful to copy your
-API key as you'll not be allowed to get it later without generating a new one.
+In order to retrieve the **SHA-OUT** key, log into your Ogone account, go to
+:menuselection:`Configuration --> Technical --> Information --> Transaction feedback --> All
+transaction submission modes`, and get or generate your API Key and Client Key. Be careful to copy
+your API key as you’ll not be
 
-.. important::
-   If you are trying Ogone as a test, with the Test Account, change the **State** to *Test Mode*. We
-   recommend doing this on a test Odoo database, rather than on your main database.
+When done, head to :menuselection:`Configuration --> Technical Information --> Transaction Feedback`
+and check the following options:
 
-Configuration on Ogone
-======================
+- :guilabel:`I would like to receive transaction feedback parameters on the redirection URLs`:
+  should be checked.
+- :guilabel:`Direct HTTP server-to-server request`: should to be set to `Online but switch to a
+  deferred request when the online request fails`.
+- Both **URL** fields should contain the same following URL, with `example` replaced by your
+  database: `https://<example>/payment/ogone/return`.
 
-Now that Odoo can communicate with Ogone, we need to make sure that Ogone can send information to
-your database.
+- :guilabel:`Dynamic eCommerce Parameters` should contain the following values: `ALIAS`, `AMOUNT`,
+  `CARDNO`, `CN`, `CURRENCY`, `IP`, `NCERROR` `ORDERID`, `PAYID`, `PM`, `STATUS`, `TRXDATE`. Other
+  parameters can be included (if you have another integration with Ogone that requires them), but
+  are not advised. The :guilabel:`URL` fields for :guilabel:`HTTP redirection in the browser` can be
+  left empty, as Odoo will specify these URLs for every transaction request.
+- In the :guilabel:`All transaction submission modes` section, fill out **SHA-OUT** passphrase and
+  disable `HTTP request for status change`.
 
-To do so, log into your Ogone account and go to :menuselection:`Configuration --> Technical
-Information --> Transaction feedback --> Direct HTTP server-to-server request`.
+If you wish to allow users to save their cards for future payments (tokenization), go to
+:menuselection:`Configuration --> Alias` and set up the tokenization process according to your
+needs.
```

Comment:
https://github.com/odoo/documentation/pull/2117#discussion_r904814169 was also wrongly resolved.

How would one know what they need to check on Ogone's side? There are many options and most admins don't know what "tokenization" actually means.

Reply:
The menus on Ogone's side are rather straightforward. Maybe I am missing something?

I still added the related info in the PR.
![Ogone](https://user-images.githubusercontent.com/104567387/175931504-0d7d300d-083f-40ad-bc4a-87a8ca066a0e.png)
.

Reply:
I'm not even sure we use any of those options. It could be worth trying to save a card on runbot while playing with the options on Ogone's side to see if they actually do something. For example, I don't believe we use their Hosted Tokenization Page.

Reply:
![hello](https://user-images.githubusercontent.com/104567387/176111783-881413ac-30c5-475a-aa21-72ab55b232a0.png)

I asked Dama about it and here's his answer. I couldn't test it because I got an error message everytime I tried to pay for something on an Odoo DB.

Reply:
Trying different settings didn't change anything it seems, I will remove that paragraph.